sdk/communication/communication-call-automation/src/models/events.ts

```diff
@@ -108,6 +110,42 @@ export interface AddParticipantFailed
   kind: "AddParticipantFailed";
 }
 
+/** The participant successfully removed event. */
```

Comment:
There has been update to how events are extended & its required values are declared - see other events for details